src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFCommand.groovy

```diff
@@ -123,46 +118,47 @@ class LSFCommand extends SubmissionCommand {
     // TODO Code duplication with PBSCommand. Check also DirectSynchronousCommand.
     /**
      * Compose the -env parameter of bsub. This supports the 'none' and 'all' parameters to clean the bsub environment or to copy the full
-     * environment during submission to the execution host. This depends on whether the `copyExecutionEnvironment` field is set (default=true). Also
+     * environment during submission to the execution host. This depends on whether the `passEnvironment` field is set (default=true). Also
      * copying specific variables (just use variable name pointing to null value in the `parameters` hash) and setting to a specific value are
      * supported.
      *
      * Note that variable quoting is left to the client code. The whole -env parameter-value is quoted with ".
      *
-     * * @return    a String of '-env "[all|none](, varName[=varValue](, varName[=varValue])*)?"'
+     * * @return    a String of '-env "[all|none|](, varName[=varValue](, varName[=varValue])*)?"'
      */
     @Override
     String assembleVariableExportString() {
-        StringBuilder qsubCall = new StringBuilder()
-        qsubCall << "-env " << "\""
-        if (copyExecutionEnvironment) {
-            qsubCall << "all"
-        } else {
-            qsubCall << "none"
-        }
+        List<String> environmentStrings = new LinkedList<>()
+
+        if (passLocalEnvironment.isPresent()) {
+            if (passLocalEnvironment.get()) {
```

Comment:
environmentStrings << passLocalEnvironment.get() ? "all" : "none" ?

Reply:
No, the idea actually was to have three states. I know it's kind of silly here, because the default if no such string is used is "all", but given that there is a tight restriction on the length of the string, I kept the code after discovering this default.